packages/opentelemetry-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/core",
-  "version": "0.25.0",
+  "version": "1.0.0",
   "description": "OpenTelemetry Core provides default and no-op implementations of the OpenTelemetry types for trace and metrics",
```

Comment:
and metrics??
As metrics is not yet released, do we want a different description for the RC

Reply:
This description is completely outdated anyway. I'll update it since it doesn't provide no-ops or defaults. It's really just a util library at this point.